sw/airborne/firmwares/rotorcraft/autopilot_arming_yaw.h

```diff
@@ -56,21 +61,25 @@ enum arming_state {
 
 uint32_t autopilot_motors_on_counter;
 enum arming_state autopilot_check_motor_status;
+float motor_kill_time;
 
 
 static inline void autopilot_arming_init(void)
 {
   autopilot_motors_on_counter = 0;
   autopilot_check_motor_status = STATUS_INITIALISE_RC;
+  motor_kill_time = -MOTOR_RE_ARM_TIME;
```

Comment:
I don't think this is necessary, it can be initialized to 0

Reply:
Hypothetically, someone could power their drone, switch to kill mode and unkill and it could 're'-arm. Probably it is unlikely, but It also doesn't hurt to have this protection right? I added it after a suggestion from @dewagter.

Reply:
That is the point of the second comment. The only way to enter the state `STATUS_MOTORS_RC_KILLED` is from arming_set (which sets the timer) or when in kill mode + kill switch on. So if you also set the timer in the second case, you can only execute the state `RC_KILLED` with a valid time, even after startup.

Reply:
Thanks for explaining. Since `autopilot_arming_check_motors_on` is called repeatedly, setting the `motor_kill_time` there is inconvenient. I now left only one route to `STATUS_MOTORS_RC_KILLED`, via `autopilot_arming_set`. This should guarantee the same safety.